solidity/test/BancorNetwork.js

```diff
@@ -599,6 +599,104 @@ contract('BancorNetwork', accounts => {
         }
     });
 
+    it('verifies that getReturnByPath return the correct amount for buying smart token', async () => {
```

Comment:
`verifies that getReturnByPath returns` (missed the 's')
`for buying the smart token` (missed the 'the')